src/views/pacbio/PacbioLibraryIndex.vue

```diff
@@ -113,6 +113,12 @@ export default {
         { key: 'selected', label: '' },
         { key: 'pool.id', label: 'pool ID', sortable: true, tdClass: 'pool-id' },
         { key: 'id', label: 'Library ID', sortable: true, tdClass: 'library-id' },
+        {
+          key: 'run_suitability',
+          label: 'Ready',
+          formatter: ({ ready_for_run }) => (ready_for_run ? '✓' : ''),
+          sortable: true,
+        },
```

Comment:
I considered trying to also add a more-details button, to see the actual errors, but space is at a premium, and we are already displaying all the required fields, so it is a bit redundant. I'll see what feedback we get.